app/components/arraySummary/ArraySummaryDataItems.tsx

```diff
@@ -41,8 +41,7 @@ const ArraySummaryDataItems = ({
   const heading = interpolateSerializableObject(
     translations[`${category}.label.heading`] ?? "",
     {
-      indexPerson: (itemIndex + WEITERE_PERSONEN_START_INDEX).toString(),
-      indexArray: (itemIndex + 1).toString(),
+      indexArray: ((customStartDisplayIndex ?? 1) + itemIndex).toString(),
```

Comment:
maybe we can keep the logic as before:

```suggestion
      indexArray: (itemIndex + (customStartDisplayIndex ?? 1)).toString(),
```

Reply:
since we are writing the code from left to right and `customStartDisplayIndex` is optional, my brain can think better if the optional variable lives on the left side :D but if you like I can revert